src/main/java/org/jenkinsci/plugins/fluentd/LoggerHolder.java

```diff
@@ -0,0 +1,25 @@
+package org.jenkinsci.plugins.fluentd;
+
+import org.fluentd.logger.FluentLogger;
+
+import java.util.logging.Logger;
+
+/**
+ * Holds the reference to the current logger
+ */
+final class LoggerHolder {
```

Comment:
As it's not strictly related to the Gerrit functionality, I believe the LoggerHolder related changes should go to a separate preceding commit.

Reply:
In this case it would not be possible to test Gerrit functionality.